README.md

```diff
@@ -16,6 +16,7 @@ Table of Contents
       * [Persistence](#persistence)
          * [Minikube](#minikube)
       * [Service Account](#service-account)
+      * [Control random admin password generation](#control-random-admin-password-generation)
```

Comment:
```suggestion
      * [Control Random Admin Password Generation](#control-random-admin-password-generation)
```

We should capitalize this one in order to maintain consistency with the other headers.